DP02_16_Galaxy_Cluster_Weak_Lensing.ipynb

```diff
@@ -0,0 +1,1566 @@
+{
```

Comment:
<p>Suggested: </p><p> </p><p>"Examine the distributions of tangential and cross ellipticities. Both exhibit Gaussian-like distributions; however, the tangential ellipticity has a mean slightly above zero, while the cross ellipticity has a mean close to zero (Kaiser 1995; Umetsu 2020)."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>adjusted </p>